pkg/gql/query/types.go

```diff
@@ -171,6 +175,36 @@ var Hex = graphql.NewScalar(graphql.ScalarConfig{
 	},
 })
 
+// Base64 is the graphql object representing a base64 scalar.
+var Base64 = graphql.NewScalar(graphql.ScalarConfig{
+	Name:        "Base64",
+	Description: "Base64 scalar type represents a byte array",
+	// Serialize serializes `CustomID` to string.
+	Serialize: func(value interface{}) interface{} {
+		switch value := value.(type) {
+		case []byte:
+			return b64.StdEncoding.EncodeToString(value)
+		default:
+			return nil
+		}
+	},
+	// ParseValue parses GraphQL variables from `string` to `[]byte`.
+	ParseValue: func(value interface{}) interface{} {
+		switch value := value.(type) {
+		case string:
+			bytes, _ := b64.StdEncoding.DecodeString(value)
```

Comment:
If we can't return the err, maybe we can just log it as an error.

Reply:
Removed as we don't need to parse it at all